docs/guide/personalization/developer_guide/attributes_search_elastic.md

```diff
@@ -0,0 +1,45 @@
+# Attributes search in Elasticsearch database
+
+If you use external data source, it is not possible to display external content in the Universal Discovery Widget as it is deprecated. Attributes search is used in scenario preview and editorials models instead.
```

Comment:
```suggestion
If you use external data source for personalization data, it is not possible to display external content in the Universal Discovery Widget as it is deprecated. Attributes search is used in scenario preview and editorials models instead.
```
Despite this being in the Perso section, I would drop "personalization" into the introductory sentence as added clue.
Also "as it is deprecated" is unclear - is UDW deprecated, or using external data?

```suggestion
If you use external data source for personalization data, it is not possible to display external content in the Universal Discovery Widget as it is deprecated. Attribute search is used in scenario preview and editorials models instead.
```